qemu/tests/cfg/blockdev_mirror_error.cfg

```diff
@@ -40,6 +40,9 @@
         image_create_support_data1 = no
     iscsi_direct:
         lun_data1 = 1
+    ceph:
+        enable_ceph_mirror1 = no
```

Comment:
remove this line for we've defined it at line58

Reply:
Have removed it, thanks zhenchao.